src/components/Nav/index.tsx

```diff
@@ -83,7 +85,17 @@ const Nav: FC<IProps> = ({ path }) => {
               />
               <HStack spacing={2} hideBelow="lg">
                 <IconButton
-                  icon={isDarkTheme ? <MdWbSunny /> : <MdBrightness2 />}
+                  icon={
+                    isDarkTheme ? (
+                      <MdWbSunny
+                        style={{ transition: "transform 0.5s, color 0.2s" }}
+                      />
+                    ) : (
+                      <MdBrightness2
+                        style={{ transition: "transform 0.5s, color 0.2s" }}
+                      />
+                    )
+                  }
```

Comment:
Can simply supply `transition` as a prop to `IconButton`
```suggestion                  
                  transition="transform 0.5s, color 0.2s"
                  icon={isDarkTheme ? <MdWbSunny /> : <MdBrightness2 />}
```